DB Setup/createDB.sql

```diff
@@ -97,3 +97,46 @@ create table Appointments (
 	PRIMARY KEY (AppointmentID)
 );
 
+CREATE OR REPLACE VIEW appointmentaummary AS 
```

Comment:
misspelled

Reply:
I think I jacked up my branch commit history. Pretty sure this directory got deleted in a ticket that got merged already.